.ci/jenkins_job_build.sh

```diff
@@ -152,6 +152,17 @@ case "${CI_JOB}" in
 	# Used by runtime makefile to enable option on intall
 	export DEFSANDBOXCGROUPONLY=true
 	;;
+"CLOUD-HYPERVISOR")
+	export CRIO="no"
+	export CRI_CONTAINERD="yes"
+	export CRI_RUNTIME="containerd"
+	export KATA_HYPERVISOR="cloud-hypervisor"
+	export KUBERNETES="yes"
+	export OPENSHIFT="no"
+	export TEST_CRIO="false"
```

Comment:
this is by default false in the crio test script, so should be fine if we remove it

Reply:
I prefer to keep it explicit it helps know what is actually being tested for the CI_JOB

Reply:
ok, agree 👍